ragstack/colbert/colbert_embedding.py

```diff
@@ -54,11 +54,20 @@ class ColbertTokenEmbeddings(TokenEmbeddings):
     encoder: CollectionEncoder
     query_tokenizer: QueryTokenizer
 
+    '''
+    checkpoint is the where the ColBERT model can be specified or downloaded from huggingface
```

Comment:
is there a better name than `checkpoint`? maybe `colbert_model_url`?

Reply:
The reason uses checkpoint is because the ColBERT library uses the same name. I added another argument called colbert_model_url that can overwrite `checkpoint` if it exists. Is that ok?